flink-connectors/flink-connector-files/src/test/java/org/apache/flink/connector/file/sink/FileSinkSpeculativeITCase.java

```diff
@@ -100,14 +96,6 @@ void testFileSinkSpeculative()
             miniCluster.executeJobBlocking(jobGraph);
```

Comment:
It's better to use `miniClusterExtension` and execute the job direct with `StreamExecutionEnvironment#execute(...)`, instead of manually maintaining and using a `MiniCluster`. A reference can be `RateLimitedSourceReaderITCase`.

Reply:
The current way is also acceptable to me, except that the `@ExtendWith(MiniClusterExtension.class)` should be removed if we choose this way.

Reply:
> The current way is also acceptable to me, except that the `@ExtendWith(MiniClusterExtension.class)` should be removed if we choose this way.
If  we remove @ExtendWith(MiniClusterExtension.class) and manually using a MiniCluster,  the ITCaseRules::ITCASE_USE_MINICLUSTER rule will be violated. Do you have other better way?

Reply:
Would it work by declaring the MiniClusterExtension as below?
```
    @RegisterExtension
    private static final MiniClusterExtension MINI_CLUSTER =
            new MiniClusterExtension(
                    new MiniClusterResourceConfiguration.Builder()
                            .setConfiguration(configure())
                            .setNumberTaskManagers(1)
                            .setNumberSlotsPerTaskManager(4)
                            .build());
```

Reply:
Sorry, I just mixed up the configuration of the cluster and the job. The cluster is started according to the MiniClusterResourceConfiguration. It will work according to your suggestion. Thank you.